stable/database/templates/cronjob.yaml

```diff
@@ -13,8 +13,13 @@
 {{- range $i, $backupGroup := $backupGroups -}}
   {{- with $globalScope }}
     {{ $backupGroupLabels := include "hotcopy.group.labels" (merge (dict "backupGroup" $backupGroup) .) }}
+{{- if .Capabilities.APIVersions.Has "batch/v1" -}}
+ {{- $cronJobVersion := "batch/v1" -}}
+{{- else -}}
+ {{- $cronJobVersion := "batch/v1beta1" -}}
+{{- end }}
```

Comment:
This isn't quite right. The `batch/v1` API version can exist without the `batch/v1/CronJob` kind existing. I already have a PR that addresses this: https://github.com/nuodb/nuodb-helm-charts/pull/288